spec/transaction_repository_spec.rb

```diff
@@ -0,0 +1,81 @@
+require_relative '../lib/transaction_repository'
+require_relative '../lib/transaction'
+
+RSpec.describe TransactionRepository do
+  before(:each) do
+    @repo = TransactionRepository.new( './spec/fixtures/mock_transactions.csv')
+  end
+
+  it 'exists' do
+    expect(@repo).to be_a(TransactionRepository)
+  end
+
+  it 'can create transaction instances' do
+    expect(@repo.all.length).to eq(5)
```

Comment:
Should we add the `.each` to this in order to iterate through `.all` and confirm that transaction is an instance of Transaction?

Reply:
sounds good.